Add "float" package to avoid "Unknown float option 'H'" error

Obtained from Debian

--- documentation/tutorials/start/start.tex.orig	2014-08-02 23:38:00 UTC
+++ documentation/tutorials/start/start.tex
@@ -9,7 +9,7 @@
 %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \documentclass{article}
-\usepackage{charter,setspace,fancybox}
+\usepackage{charter,setspace,fancybox,float}
 \textwidth 15cm
 \textheight 23cm
 \oddsidemargin  +0.75cm
